pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/stats/SubscriptionStatsImpl.java

```diff
@@ -115,9 +116,11 @@ public class SubscriptionStatsImpl implements SubscriptionStats {
     public List<ConsumerStatsImpl> consumers;
 
     /** Tells whether this subscription is durable or ephemeral (eg.: from a reader). */
+    @JsonProperty("isDurable")
     public boolean isDurable;
 
     /** Mark that the subscription state is kept in sync across different regions. */
+    @JsonProperty("isReplicated")
     public boolean isReplicated;
```

Comment:
I think that the properties with `is` prefix shouldn't ever end up in the JSON output. We should address that problem instead and keep `durable` and `replicated` in the output instead of making them `isDurable` and `isReplicated`.

Reply:
+1 with lari

Reply:
Thanks for the suggestion, I has been modified and it is a breaking change.